File: solvers.julia/IOD/inidata_num_IOD.jl
function inidata_num_IOD()

    data = Vector{Matrix{Float64}}()
    push!(data,randn(4,2))
    push!(data,randn(4,2))
    push!(data,randn(4,2))
    push!(data,randn(4,2))
    push!(data,randn(4,2))
    return data

end

File: solvers.python/py_IOD/inidata_num_IOD.py
import numpy as np

def inidata_num_IOD():
    data = [None]*5
    data[0] = np.random.randn(4,2)
    data[1] = np.random.randn(4,2)
    data[2] = np.random.randn(4,2)
    data[3] = np.random.randn(4,2)
    data[4] = np.random.randn(4,2)
    return data